app/src/main/res/resource/values-de/strings.xml

```diff
@@ -142,7 +144,7 @@
     <string name="feedback_name_is_required">Name ist nötig</string>
     <string name="feedback_sure">Sicher?</string>
 
-    <string name="feedback_notify_set_back_to_top">Sie können den Rücktasteneffekt einstelleln</string>
+    <string name="feedback_notify_set_back_to_top">Sie das Verhalten der "Zurück" Taste ändern</string>
```

Comment:
Now this is no sentence anymore. Note that the English original even has a dot `.` at the end, so you should make this a sentence here and also include the trailing dot.